### ISSUE TITLE
Evaluation of Volumetric Model

### ISSUE THREAD

**Issue by sunwonlikeyou** [karfly/learnable-triangulation-pytorch#123]
I simulated pretrained volumetric model with pretrained backbone.
and i got these result.
'S11': {'Average': 158.4710789766407, 'Directions': 13.81251708984375, 'Discussion': 20.87702763545049, 'Eating': 107.31265129841549, 'Greeting': 104.50263323102679, 'Phoning': 249.43584526909723, 'Posing': 33.8868872601053, 'Purchases': 160.68772977941177, 'Sitting': 87.17900390625, 'SittingDown': 264.5645211641906, 'Smoking': 240.20394674161585, 'TakingPhoto': 190.77657277960526, 'Waiting': 58.71925354003906, 'Walking': 307.15990271935095, 'WalkingDog': 243.03764997209822, 'WalkingTogether': 201.88863716873468} 

and also i checked heatmap from backbone in volumetric model
![image](https://user-images.githubusercontent.com/54057384/107490797-b4293800-6bcd-11eb-8def-1c9ee2499e47.png)

but heatmap from backbone in algebraic model is
![image](https://user-images.githubusercontent.com/54057384/107491250-3ca7d880-6bce-11eb-9645-7013c366ab38.png)

is there problem or mistake on evaluation??

**Comment by shrubb** [comment]
Looks like you caught a bug. These heatmaps seem unnatural to me, as it the network is underfit or overfit. For example, I believe there shouldn't be such checkerboard patterns. You probably messed up `.requires_grad`, `.eval()` vs `.train()` or something like that.

**Comment by sunwonlikeyou** [comment]
Backbones were all eval() and also all tensors' requires_grad were also False. 
And That plottings are heatmap of 'Directions-1' which shows good quantitative result but i can check poor heatmap result.
And also I checked Loading pretrained weights for backbone and whole model.
can you show me your log files??

**Comment by shrubb** [comment]
Hi, unfortunately I don't have logs :( I still bet you have a bug somewhere in the backbone. I suggest that you try checking activations and gradients layer-wise.
